llvm_util/llvm2alive.cpp

```diff
@@ -711,14 +712,47 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
           }
         } else if (name == "align") {
           llvm::Value *ptr = bundle.Inputs[0].get();
-          llvm::Value *align = bundle.Inputs[1].get();
-          if (bundle.Inputs.size() != 2)
-            // TODO: "align" assume operand bundle may take a third argument.
+          auto *aptr = get_operand(ptr);
+          if (!aptr)
             return error(i);
 
-          auto *aptr = get_operand(ptr), *aalign = get_operand(align);
-          if (!aptr || !aalign)
+          auto *align = llvm::cast<llvm::ConstantInt>(bundle.Inputs[1].get());
+          if (align->uge(UINT64_MAX))
             return error(i);
+          uint64_t align_value = align->getZExtValue();
+          auto *aalign = make_intconst(align_value, 64);
```

Comment:
I found that the existing code crashes when giving >64 bit constant for the first value:
```
// Assume::toSMT
   2489	  case Align: {
   2490	    // assume(ptr, align)
   2491	    const auto &vptr = s.getAndAddPoisonUB(*args[0]);
-> 2492	    uint64_t align = *dynamic_cast<IntConst *>(args[1])->getInt(); // getInt() crashes
```
I fixed the bitwidth to 64 to make `getInt()` succeed. This is okay because a constant larger than UINT64_MAX is guarded by the `if(..uge())` above.